### PR TITLE
Add RNA with UMIs README.md

### PR DESCRIPTION
**Pull request by kayleemathews**
Adding RNA with UMIs overview doc (README.md)

I'd specifically like feedback on the workflow diagram in the very beginning of the doc. Do you think it's helpful? What would make it more helpful (task names vs. tool names, detailed workflow vs. higher level overview, etc.)?

### CODE REVIEW

File: website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+| [tasks.rnaseqc2](https://github.com/broadinstitute/warp/blob/develop/tasks/broad/RNAWithUMIsTasks.wdl) | rnaseqc | [RNA-SeQC](https://github.com/getzlab/rnaseqc) | Uses the genome-aligned, duplicate-marked BAM file to calculate TPMs, gene counts, exon counts, fragment sizes, and additional metrics, each of which is outputted to an individual file. |
+| [tasks.CollectRNASeqMetrics](https://github.com/broadinstitute/warp/blob/develop/tasks/broad/RNAWithUMIsTasks.wdl) | CollectRNASeqMetrics | [Picard](https://broadinstitute.github.io/picard/) | Calculates RNA metrics; strand specificity is set to SECOND_READ_TRANSCRIPTION_STRAND. |
+| [tasks.CollectMultipleMetrics](https://github.com/broadinstitute/warp/blob/develop/tasks/broad/RNAWithUMIsTasks.wdl) | CollectMultipleMetrics | [Picard](https://broadinstitute.github.io/picard/) | Collects multiple classes of metrics; runs tools CollectInsertSizeMetrics and CollectAlignmentSummaryMetrics. |
+| [tasks.MergeMetrics](https://github.com/broadinstitute/warp/blob/develop/tasks/broad/RNAWithUMIsTasks.wdl) | --- | --- | Merges the metrics from Picard and RNA-SeQC. |
```
Comment (gbggrant) [inline website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md:126]: I have moved MergeMetrics to BroadInternalRNAwithUMIs as this is a task used to make data ready for TDR.

File: website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+
+#### 5. Gene quantification
+
+After duplicate reads have been tagged, the workflow uses [RNA-SeQC](https://github.com/getzlab/rnaseqc) to quantify the expression level of transcripts based on the number of reads that align to one or more exons of each gene (`rnaseqc2_gene_counts`). Exon-level expression is also quantified based on the number of reads that align to each exon (`rnaseqc2_exon_counts`). 
+
+#### 6. Metric calculation
+
+The pipeline uses [RNA-SeQC](https://github.com/getzlab/rnaseqc), Picard’s [CollectRNASeqMetrics](https://gatk.broadinstitute.org/hc/en-us/articles/360037057492-CollectRnaSeqMetrics-Picard-), and Picard’s [CollectMultipleMetrics](https://gatk.broadinstitute.org/hc/en-us/articles/360037594031-CollectMultipleMetrics-Picard-) to calculate summary metrics that can be used to assess the quality of the data each time the pipeline is run. The resulting metrics are output to several individual files and merged together to create the `MergeMetrics.unified_metrics` output file.
```
Comment (gbggrant) [inline website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md:164]: I have moved MergeMetrics to BroadInternalRNAwithUMIs as this is a task used to make data ready for TDR.

File: website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+| picard_rna_metrics | Metrics file containing the output of Picard’s CollectRnaSeqMetrics tool. | TXT |
+| picard_alignment_summary_metrics | Metrics file containing output of Picard’s CollectAlignmentSummaryMetrics tool. | TXT |
+| picard_insert_size_metrics | Metrics file containing output of Picard’s CollectInsertSizeMetrics tool. | TXT |
+| picard_insert_size_histogram | Histogram chart of insert size. | PDF |
+| picard_base_distribution_by_cycle_metrics | Metrics file containing the output of Picard’s CollectBaseDistributionByCycle tool. | TXT |
+| picard_base_distribution_by_cycle_pdf | Chart of nucleotide distribution per cycle. | PDF |
+| picard_quality_by_cycle_metrics | Metrics file containing the output of Picard’s MeanQualityByCycle tool. | TXT |
+| picard_quality_by_cycle_pdf | Chart of mean quality by cycle. | PDF |
+| picard_quality_distribution_metrics | Metrics file containing the output of Picard’s QualityScoreDistribution tool.
+| unified_metrics | Merged metrics file containing RNA-SeQC, Picard RNA-seq, Picard insert size, and Picard alignment metrics. | TXT |
```
Comment (gbggrant) [inline website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md:193]: unified_metrics has been moved to BroadInternalRNAwithUMIs.

**Review by gbggrant — APPROVED** [review]
Looks good. Two comments on diagram:
1) As mentioned elsewhere in the review, I've moved the MergeMetrics task out of RNAWithUMIs (and moved it into BroadInternalRNAWithUMIs).
2) There's actually a 'CopyReadGroupsToHeader' task that runs *after* STAR to create the Transcriptome-aligned BAM. I'm not sure if you intentionally omitted that for clarity.

**Review by nikellepetrillo — APPROVED** [review]
Only comment I have is that MergeMetrics has been moved out of here and into BroadInternalRNAWithUMIs pipelines. Other than that it looks good!

File: website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+![RNAWithUMIs_diagram](RNAWithUMIs_diagram.png)
+
+## Introduction to the RNA with UMIs workflow
+
+The [RNA with UMIs pipeline](https://github.com/broadinstitute/warp/blob/develop/pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl) is an open-source, cloud-optimized workflow for processing total RNA isolated with the Transcriptome Capture (TCap) method. TCap is a technique that hybridizes exome baits to cDNA library preparations to better facilitate RNA-sequencing in low-input or poor quality (degraded) samples. These libraries may additionally be prepared with Unique Molecular Identifiers (UMIs) which can help distinguish biological signal from noise resulting from PCR amplification. 
+
+Overall, the workflow performs UMI correction, aligns reads to the genome, quantifies gene counts, and calculates quality metrics. The workflow produces genome- and transcriptome-aligned BAMs with indices and a merged quality metrics file.
```
Comment (nikellepetrillo) [inline website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md:17]: No longer outputting 1 merged metrics file

File: website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md (hunk 1 above)
Comment (kayleemathews) [reply website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md]: Removed MergeMetrics

File: website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md (hunk 2 above)
Comment (kayleemathews) [reply website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md]: Updated

File: website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md (hunk 3 above)
Comment (kayleemathews) [reply website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md]: updated

File: website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md (hunk 4 above)
Comment (kayleemathews) [reply website/docs/Pipelines/RNA_with_UMIs_Pipeline/README.md]: updated - thanks!

**Pull request merged**